pkg/backupapi/recovery_point.go

```diff
@@ -18,3 +30,61 @@ type RecoveryPoint struct {
 	CreatedAt         string `json:"created_at"`
 	UpdatedAt         string `json:"updated_at"`
 }
+
+// CreateRecoveryPointRequest represents a request to create a recovery point.
+type CreateRecoveryPointRequest struct {
+	PolicyID string `json:"policy_id"`
+}
+
+// UpdateRecoveryPointRequest represents a request to update a recovery point.
+type UpdateRecoveryPointRequest struct {
+	Status string `json:"status"`
```

Comment:
As the API documents, we need to put recovery point ID in the body. I will ask @ducpx  if we can remove it from the body. Because the recovery point ID is in the URL path.

Reply:
It's for create, not path.

Reply:
@ducpx Could you give some thoughts on this?